.github/workflows/ci-label-check.yml

```
@@ -16,7 +16,9 @@ jobs:
     steps:
       - name: Check PR author
         id: check_author
-        run: echo "::set-output name=is_dependabot::$(echo ${{ github.event.pull_request.user.login }} | grep -o 'dependabot')"
+        run: |
+          is_dependabot=$(echo ${{ github.event.pull_request.user.login }} | grep -o 'dependabot')
+          echo "is_dependabot=$is_dependabot" >> $GITHUB_OUTPUT
 
       - name: Check PR label
         if: steps.check_author.outputs.is_dependabot != 'dependabot'
```

wonder why we couldn't just do this here:
```suggestion
        if: github.event.pull_request.user.login != 'dependabot'
```

---

This seems much better IMO rather than `grep`ing manually.